packages/cli/src/util/telemetry/commands/dns/add.ts

```diff
@@ -0,0 +1,36 @@
+import { TelemetryClient } from '../..';
+
+export class DnsAddTelemetryClient extends TelemetryClient {
+  trackCliArgumentDomain(domain?: string) {
+    if (domain) {
+      this.trackCliArgument({
+        arg: 'domain',
+        value: this.redactedValue,
+      });
+    }
+  }
+  trackCliArgumentName(name?: string) {
+    if (name) {
+      this.trackCliArgument({
+        arg: 'name',
+        value: this.redactedValue,
+      });
+    }
+  }
+  trackCliArgumentType(type?: string) {
+    if (type) {
+      this.trackCliArgument({
+        arg: 'type',
+        value: type,
```

Comment:
I would be tempted to filter these to that known list, in case they mistype a type or miss an argument and get sent something that would otherwise be redacted.